packages/api/src/external/commonwell/patient.ts

```diff
@@ -385,6 +385,12 @@ async function updatePatientAndLinksInCw(
       getOrgIdExcludeList
     );
 
+    setPatientDiscoveryStatus({
+      patientId: patient.id,
+      cxId: patient.cxId,
+      status: "completed",
+    });
+
```

Comment:
For create there is `getStoreIdsAndStatusFn` that sets the status to completed but when updating we only call this when there is an issue so need to add completed at the end if all goes well